upgrading.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading from v5.7.0 or earlier
+
+From v6.0.0 of the users cookbook we have removed the hard dependency on data bags. This means that you can no longer pass the data bag name to the users_manage resource.
+
+The resource now expects an Array of Hashes for all the users and can as such be used with other sources then data bags.
+
+You can still use data bags as detailed below.
+
+## Major Changes
+
+Data bags are no longer directly supported. Before v6.0.0 you would do the following:
```

Comment:
Why does this change have to be made? Can't we keep this functionality in the resource while also having the new functionality? This is a *really* big change for this cookbook and will impact the users quite a bit, even with a major version bump.

Reply:
I discussed this briefly in the sous-chefs channel and the consensus was that it was better to make it independent of the data source. 

I think adding one line of code to load the data bag is not too bad for a major version upgrade right?

If you disagree I can refactor it to make a source or a data bag possible. But I personally do think this way is cleaner and it does less chef searches as well if you have multiple invocations of the resource.

Reply:
Ah yes, I didn't get a chance to read the backlog. That makes sense.